app/javascript/src/QuickStarts/templates/template.yaml

```diff
@@ -80,20 +100,102 @@ spec:
+
+      # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Did you complete the task successfully?
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+      # optional - the task's success and failure messages
+      summary:
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+    - title: Promote configuration to staging
+      description: |-
+        Before you can send a test request to your API product, you must promote the APIcast configuration to staging and production. An APIcast configuration consists of the backends you have added to a product with the corresponding mapping rules, applications, and application plans.
+
+        ### Procedure
+
+        1. Go to <strong>[Your\_product\_name] > Integration > Configuration</strong> page.
+              - Click the context selector on the header and select _Product_.
+              - Click <strong>[Your\_product\_name]</strong> and go to the _Overview_ page.
+              - Click <strong>Integration > Configuration</strong>  on left navigation.
+        1. Under _APIcast Configuration_, click <strong>Promote v.[n] to Staging APIcast</strong>.
+        1. Under Staging APIcast, click <strong>Promote v.[n] to Production APIcast</strong>.
+
+        > <strong>Tip:</strong> Every time you modify the product configuration and before making calls to your API, you must promote the updated configuration to the staging and production environments.
+
+      # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Did you complete the task successfully?
+        failedTaskHelp: This task isn’t verified yet. Try the task again.
+      # optional - the task's success and failure messages
+      summary:
+        success: Shows a success message in the task header
+        failed: Shows a failed message in the task header
+    - title: Test example curl
+      description: |-
+        After promoting the API cast configuration, you can send a command line request to a product to test the integration of a backend based on the first mapping rule added to the product.
+
+        ### Procedure
+
+        - To test requests to your API product, copy the command provided in Example curl for testing and run it in a terminal. 
+        - The curl command example is based on the first mapping rule of the product. 
+        - After you run the command, you should get an HTML response containing results from the backend you are testing. 
+        - If you do not get a response, delete the catch-all mapping rule from your product, promote the new APIcast configuration to staging and then to production, and run the example curl command.
+
+      # optional - the task's Check your work module
+      review:
+        instructions: |-
+          - Did you complete the task successfully?
```

Comment:
change to: 

- Did you create an application plan for your product?
- Did you create an application that subscribes to the application plan?
-Did you promote the APIcast configuration to staging and production?